scripts/lib/couchdb.sh

```
@@ -0,0 +1,36 @@
+# CouchDB init helpers for ndb-setup scripts.
+# Source after common.sh. All helpers operate on the instance dir in $path and read credentials from its .env.
+#
+# The database-only CouchDB runs as the container "${INSTANCE_NAME}-db-entrypoint". That name is reused by
+# the replication-backend in the permission profiles, so the init container must be removed (couchdbInitStop)
+# before switching profiles. Instance data lives in ./couchdb/data and survives the container removal.
+
+# Start the database-only CouchDB and wait until it answers on /_up.
+# Requires: $path. Sets globals: DB_CONTAINER, DB_LOCAL_URL, DB_USER, DB_PASSWORD.
+couchdbInitStart() {
+  DB_LOCAL_URL="http://127.0.0.1:5984"
+  DB_USER=$(getVar "$path/.env" COUCHDB_USER)
```

Shellcheck (0.11.0)

[warning] path is referenced but not assigned.

(SC2154)